cmd/daprd/main.go

```diff
@@ -108,15 +108,14 @@ import (
 )
 
 var log = logger.NewLogger("dapr.runtime")
```

Comment:
I would use the below: 

var (
    log = logger.NewLogger("dapr.runtime")
    logContrib = logger.NewLogger("dapr.contrib")
)

Reply:
Thanks for the review 👍 I've made the changes